scripts/clobber_merge.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
```

Comment:
Can you add a short description of what this script does?

Reply:
Done. Missed this in my pass through to add documentation, thanks for catching :-)